packages/pf4-component-mapper/src/files/plain-text.js

```diff
@@ -4,11 +4,11 @@ import PropTypes from 'prop-types';
 import { TextContent, Text } from '@patternfly/react-core';
 import validTextFields from '@data-driven-forms/common/src/utils/valid-text-fields';
 
-const PlainText = ({ label, name, variant }) => (
-  <TextContent>
+const PlainText = ({ label, name, variant, TextContentProps, ...rest }) => (
```

Comment:
please catch 'component' prop coming from renderer, it could cause issues.

Reply:
I think this actually is closer to the current way of customization in our other mappers (stuff like required, disabled, etc.). Plus the component prop is always caught by the use field hook. And you can't really use the `component` key in the schema.

Reply:
The plaintext component will receive `component="plain_text"` and this is then passed via `rest` to the PF `Text` component. There is no `useFieldApi` hook here to catch it.

Reply:
we do not catch `component` in useFIeldApi!?

Reply:
We do, but plain text is not using it. It's not a field, it's just a component.